lodash.js

```diff
@@ -11405,6 +11424,37 @@
     }
 
     /**
+     * Updates the value at `path` of `object` with given `func`. If a portion of `path` doesn't exist
+     * it's created. Arrays are created for missing index properties while objects
+     * are created for all other missing properties.
```

Comment:
Can you add a sentence on that params the`func` expects. You can reference other methods like `_.every` for an example. Also can you rename the param `updater`.